srcpkgs/lua-lgi/template

```diff
@@ -0,0 +1,81 @@
+# Template file for 'lua-lgi'
+pkgname=lua-lgi
+version=0.9.2
+revision=4
+wrksrc="lgi-${version}"
+hostmakedepends="pkg-config"
+makedepends="lua-devel lua51-devel lua52-devel libglib-devel libffi-devel gobject-introspection gir-freedesktop"
+_depends="gir-freedesktop"
+depends="${_depends} lua"
+short_desc="Lua bindings for gnome/gobject using gobject-introspection library"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="MIT"
+homepage="https://github.com/pavouk/lgi"
+distfiles="https://github.com/pavouk/lgi/archive/$version.tar.gz"
+checksum=cfc4105482b4730b3a40097c9d9e7e35c46df2fb255370bdeb2f45a886548c4f
+
+_alternatives="
+ lgi:dump-typelib:/usr/bin/dump-typelib__luaver__
+"
+alternatives=${_alternatives/__luaver__/5.3}
+
+post_extract() {
+	mkdir -p lua5.1
+	mv * lua5.1 || true
+	cp -a lua5.1 lua5.2
+	cp -a lua5.1 lua5.3
+}
+
+do_build() {
+	for luaver in 5.1 5.2 5.3; do
+		cd ${wrksrc}/lua${luaver}
+		make LUA_VERSION="${luaver}" CC="$CC" AR="$AR" RANLIB="$RANLIB" CFLAGS="${_common_cflags} -I${XBPS_CROSS_BASE}/usr/include/lua${luaver}" ${makejobs}
+	done
+}
+
+do_install() {
+	vmkdir usr/bin
+	for luaver in 5.1 5.2 5.3; do
```

Comment:
for most of these, any reason not to just use `make install`?

Reply:
The `dump-typelib.lua` file, I change its name (based on lua version) and add to to the alternative group, make install will just move the file to `usr/bin/`